qiskit_optimization/__init__.py

```diff
@@ -22,7 +22,7 @@
 classical simulators, as well as on real quantum devices via Qiskit.
 
 This module enables easy, efficient modeling of optimization problems using `docplex
-<https://developer.ibm.com/docloud/documentation/optimization-modeling/modeling-for-python/>`_.
+<http://ibmdecisionoptimization.github.io/docplex-doc/>`_.
```

Comment:
```suggestion
<https://ibmdecisionoptimization.github.io/docplex-doc/>`_.
```
The link in the readme is to https which I think is preferred nowadays over http, so I think that should be here too

Reply:
Thanks for catching -- somehow this escaped my notice.